tests/unit/moduleapi/timer.tcl

```diff
@@ -54,7 +56,14 @@ start_server {tags {"modules"}} {
         assert_equal {} [r test.gettimer $id]
     }
 
-    test "Unload the module - timer" {
+    test "Module can be unloaded only when timer was finished" {
+        r set "timer-incr-key" 0
+        set id [r test.createtimer 1000 timer-incr-key]
```

Comment:
Maybe 100ms would make the test end faster.

Reply:
ok